### ISSUE TITLE
Question about exporting the site layout to a different site..

### ISSUE THREAD

**Issue by DjWellz** [OllieWP/ollie#128]
Hello, I was wondering what is the process required (and if it's even possible) to take a layout and design that I create on a staging site using Ollie, and then export it to a live site (from where the staging site files originated)?
In my case, the live site is an old site designed on WordPress using the Genesis theme along with a child theme, so this would NOT be going from Block theme to another Block theme. 

What I'm hoping to do is clone the live site to create the staging site and use Ollie as the new theme on the staging site. Since the original site was built using a classic theme, a lot of the plugins and widgets that are in use will be turned off. Once I get the staging site to look the way I want using Ollie, I would like to take that new layout and design and apply it to the live site. 
From the brief info on the Ollie-WP site, it sounds like it's possible to export the site design and apply it to another site. Since I'm new to all of this, I don't know if I'm misunderstanding the concept. But I'm hopeful that someone here will clear things up for me and guide me on how to make these changes (if possible). I'm aware that while working on the staging site, the live site will continue to grow since there is a weekly blog and an WooCommerce store making sales daily. For this reason, simply going live with the staging site won't be possible, since I don't want to lose any of the data that was added to the live site after the staging site was created.

To summarize it all, I'd like to do the following:

1. Clone my live site and create a staging site (on Siteground hosting).
2. Install the Ollie theme on the staging site which will completely change the look and layout of the site.
3. Fix the staging site so that it looks the way I want. 
4. Take that new look/design over to the live site, to avoid having to redesign it from scratch after installing Ollie. 

Thanks in advance,
Wells.

**Comment by tibesararchives** [comment]
Boy that's a lot of steps Mr. Wells.

If it was me, I would install the Ollie theme on the live site, click on Live Preview and look at all the critical pages.

If everything looks okay, do a backup of your live site and then Activate Ollie. 

I recently did this on a site that was running Newspack theme and it was pretty seamless. No back and forth from Live to Staging to Live as you are planning.

Just my 2-cents bro!

Marcus

**Comment by DjWellz** [comment]
Thanks Marcus, and I hear you, lots of step. The issue I'm dealing with is that the live preview is a mess with many of the items on the live site not showing up. The old site has a lot of custom code and widgets that disappear once you switch to a block theme. I've already set up the staging site and have been testing out the new layout using Ollie. Lots of changes but it's beginning to look the way I want it to look when it's finished.  The issue is that while I'm working on the staging site, the live site continues to grow, so I can't just go live with the staging site since that would cause all of the new info to be lost. Alternatively, if I could take those updates from the live site and add them to the staging site so that its current, I would do that instead. But alas, I don't know how to do that yet.

**Comment by tibesararchives** [comment]
How To Overhaul An Old Website To New Website The Right Way [5 Step Process]youtu.beRegards,Marcus TibesarOn Jan 7, 2024, at 10:20 PM, DjWellz ***@***.***> wrote:﻿
Thanks Marcus, and I hear you, lots of step. The issue I'm dealing with is that the live preview is a mess with many of the items on the live site not showing up. The old site has a lot of custom code and widgets that disappear once you switch to a block theme. I've already set up the staging site and have been testing out the new layout using Ollie. Lots of changes but it's beginning to look the way I want it to look when it's finished.  The issue is that while I'm working on the staging site, the live site continues to grow, so I can't just go live with the staging site since that would cause all of the new info to be lost. Alternatively, if I could take those updates from the live site and add them to the staging site so that its current, I would do that instead. But alas, I don't know how to do that yet.

**Comment by DjWellz** [comment]
thanks again, Marcus. That's a helpful video, but in my case, it fails to explain what to do when the live site has grown AFTER you originally created the staging site. That's the part I'm trying to figure out. Aside from new Blog posts, there is also new sales data from the WooCommerce store that is part of the site. I don't want to lose any of that by importing the staging site since I believe it would overwrite what is there. I was under the impression that with the new WordPress 6.4, we had the ability to export a website design and install it like a new theme over the old one. At least this option would allow for a quicker update without losing the new data.

**Comment by tibesararchives** [comment]
I know that Kadence has an Export in the Customizer that allows all the customization to be saved and then imported to a new or existing Kadence site. I'm not sure if Ollie offers this but, I will certainly keep my eyes open. Perhaps there is a plugin that will export Full Site Editing settings to something like a json file and then on the live site add the Ollie theme and then import the json file.

**Comment by tibesararchives** [comment]
Is this video useful? [[]](https://youtu.be/3ZY5ZQNpqnQ?si=Xj-DEhBq--BvQ7GN)(url)

**Comment by tibesararchives** [comment]
https://youtu.be/3ZY5ZQNpqnQ?si=Xj-DEhBq--BvQ7GN

**Comment by DjWellz** [comment]
This is more along the lines of what I'm looking for. I will get more
details about it, test it out and report back. Thanks for your help.
